pytest_mh/conn/ssh.py

```diff
@@ -432,7 +432,7 @@ def __init__(
         :param logger: Multihost logger.
         :type logger: MultihostLogger
         :param timeout: Timeout in seconds (defaults to 300), value
-            ``0`` means that timeout is disabled.
+            ``0`` means that timeout is disabled. 3 second minimum value.
```

Comment:
If 3 is the minimum value, then add some code to check this argument and increment it (or error) in case someone sets a lower value.